### PR TITLE
Make pmon feature delayed flag as jinja template

### PR DESCRIPTION
**Pull request by yejianquan**
#### What I did:
To fix: https://github.com/sonic-net/sonic-buildimage/issues/19126
Depends on: https://github.com/sonic-net/sonic-host-services/pull/135 and it's submodule update.

ps: this is an enhancement/fix of https://github.com/sonic-net/sonic-buildimage/pull/19482 since Abhishek is in OOF

#### Why I did it
PMON was delayed originally for faster uptime of data plane related services in case of fast/warm reboot.
This is not needed for T2/SpineRouter . Infact we need PMON to be up asap because of:

pmon need to enable asap to detect ASIC's on Supervisor.
pmon need to enable asap for bring-up of 400G ports on LC's fast because of CMIS state machine present in PMON.
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How to verify it
Manual Verification and UT has been added in https://github.com/sonic-net/sonic-host-services/pull/135

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by yejianquan** [comment]
Hi @mlok-nokia @arlakshm, please help to review

**Comment by yejianquan** [comment]
Hi @yxieca , could you please help to review and merge?

**Comment by yejianquan** [comment]
@abdosi for viz

File: files/build_templates/init_cfg.json.j2 [hunk 1]
```diff
@@ -38,7 +38,7 @@
 {%- set features = [("bgp", "{% if not DEVICE_RUNTIME_METADATA['ETHERNET_PORTS_PRESENT'] or ('CHASSIS_METADATA' in DEVICE_RUNTIME_METADATA and DEVICE_RUNTIME_METADATA['CHASSIS_METADATA']['module_type'] in ['supervisor']) %}disabled{% else %}enabled{% endif %}", false, "enabled"),
                    ("database", "always_enabled", false, "always_enabled"),
                    ("lldp", "enabled", true, "enabled"),
-                   ("pmon", "enabled", true, "enabled"),
+                   ("pmon", "enabled", "{% if 'type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['type'] == 'SpineRouter' %}False{% else %}True{% endif %}", "enabled"),
```
Comment (saiarcot895) [inline files/build_templates/init_cfg.json.j2:41]: Why use titlecase instead of lowercase here?

Comment (yejianquan) [reply files/build_templates/init_cfg.json.j2:41]: This is to align with https://github.com/sonic-net/sonic-host-services/pull/135
In this change https://github.com/sonic-net/sonic-host-services/blob/ca6b3cdb9a56604765ada4ac67234c117f0fb5ad/scripts/featured#L79 , the host-service takes titlecase only and evaluate them with python eval.

**Review by saiarcot895 — APPROVED** [review]

**Pull request merged**